Amazon.QLDB.Driver.Tests/AsyncQldbDriverTests.cs

```diff
@@ -34,15 +33,56 @@ public class AsyncQldbDriverTests
     {
         private const string TestTransactionId = "testTransactionId12345";
         private static AsyncQldbDriverBuilder builder;
-        private static Mock<AmazonQLDBSessionClient> mockClient;
+        private static MockSessionClient mockClient;
         private static readonly byte[] digest = { 89, 49, 253, 196, 209, 176, 42, 98, 35, 214, 6, 163, 93,
             141, 170, 92, 75, 218, 111, 151, 173, 49, 57, 144, 227, 72, 215, 194, 186, 93, 85, 108,
         };
 
+        private static SendCommandResponse startSessionResponse = new SendCommandResponse
+        {
+            StartSession = new StartSessionResult { SessionToken = "testToken" },
+            ResponseMetadata = new ResponseMetadata { RequestId = "testId" }
+        };
+        private static SendCommandResponse startTransactionResponse = new SendCommandResponse
+        {
+            StartTransaction = new StartTransactionResult { TransactionId = TestTransactionId },
+            ResponseMetadata = new ResponseMetadata { RequestId = "testId" }
+        };
+
+        private SendCommandResponse executeResponse(List<ValueHolder> values)
+        {
+            Page firstPage;
+            if (values == null)
```

Comment:
If values is null and we include `Values = null` into the response then we get a `ExecuteStatementResult` with a null `List<ValueHolder>` which results in an unwanted null exception.
If we don't include `Values = null` in the response, then `ExecuteStatementResult` gets a non-null empty `List<ValueHolder>` instead.

Reply:
So the setting the value = null is not equivalent to not setting it? Sort of like null vs. not present?

Reply:
Correct.

The difference amounts to this:

![image](https://user-images.githubusercontent.com/62349012/112066834-b23b8880-8b2c-11eb-93d5-9911e996ad7a.png)

vs

![image](https://user-images.githubusercontent.com/62349012/112066855-be274a80-8b2c-11eb-8382-3f18c5612773.png)

Reply:
Wait - I didn't get that. Are you saying that setting Values = null results in Values being null, while not setting is the same as setting it equal to the empty list? Or something else?

Reply:
Yes exactly. Not setting it results in the internal `List<ValueHolder>` being an empty list rather than a null list. While setting it results in a null list rather than an empty list.